### ISSUE TITLE
Problems with AGP8+ when minify enabled

### ISSUE THREAD

**Issue by pvegh** [Usercentrics/android-sample#9]
New AGP versions since 8.0 are doing more aggressive optimizations.
The current versions of UC are not tested in those environments as they can't be minified successfully, as warnings become errors since AGP 8.

Should always support the latest stable versions (currently 8.1), also keep track of betas (8.2 canary 16) so this doesn't happen at customers when under pressure.

Created a support ticket, adding it here in the hope of a faster fix, since we're blocked by this.

Thanks

**Comment by pvegh** [comment]
Repro project created by wizard, added UC with init call and enabled minify.
[usercentricsR8fail.zip](https://github.com/Usercentrics/android-sample/files/12408402/usercentricsR8fail.zip)

**Comment by uc-leo** [comment]
thanks @pvegh for reporting it, we are already looking into it.

**Comment by pvegh** [comment]
Thanks for the release 2.8.3 with the changes, it fixes the above problem for the example.
We'll test some more with the prod project and open a new issue if necessary.
